.pre-commit-config.yaml

```diff
@@ -1,4 +1,10 @@
 repos:
+  - repo: https://github.com/pre-commit/mirrors-prettier
+    rev: "v2.7.1"
+    hooks:
+      - id: prettier
+        files: '^(?!CHANGELOG.md|mlflow/pypi_package_index.json|docs/|mlflow/server/js/).+\.(md|json|ya?ml)$'
+        args: ["--no-config", "--print-width", "100"]
```

Comment:
The core change of this PR to enforce formatting.

Reply:
`--no-config` to enforce using the default configuration and make formatting deterministic.